src/Components/Web/src/Forms/InputLargeTextArea/InputLargeTextAreaJsCallbacksRelay.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    internal class InputLargeTextAreaJsCallbacksRelay : IDisposable
```

Comment:
```suggestion
    internal sealed class InputLargeTextAreaJsCallbacksRelay : IDisposable
```

Reply:
Similarly with this one - is there a reason not to have this be on `InputTextAreaLarge`? Like have `NotifyChange` be on `InputTextArea`?

Reply:
> Similarly with this one - is there a reason not to have this be on `InputTextAreaLarge`? Like have `NotifyChange` be on `InputTextArea`?

I wanted to follow a consistent pattern with our `InputFile` approach which uses a comparable relay: https://github.com/dotnet/aspnetcore/blob/main/src/Components/Web/src/Forms/InputFile/InputFileJsCallbacksRelay.cs

Reply:
I think the InputFile used that pattern because it had different implementations based on server vs Wasm. If it's not too much trouble, we could avoid the pattern here. It's an unnecessary additional allocation that doesn't really help things like testability.